### DISCUSSION TITLE
Failing CIVET test under "Build unit test"

### DISCUSSION THREAD

**Discussion by smpark7** [idaholab/moose#25406] [Q&A Tools] [answered]
I'm working on a PR to remove various deprecated code in Moltres. After replacing `ModulesApp::registerAll` with `ModulesApp::registerAllObjects<MoltresApp>` and other associated changes, the CI test for the PR fails at the "Build unit test" stage. Here's the link to the Civet test results: https://civet.inl.gov/job/1749366/

It seems to be related to the Reactor module. Do I need to make additional changes to accommodate compiling `reactor` with Moltres?

**Comment by GiudGiud** [comment]
Hello

Nothing comes to mind rn. @permcody were there specific changes for unit tests to do?

Guillaume

**Comment by permcody** [comment, marked as answer]
The easy fix is to make sure the set of modules for your Unit Makefile
exactly matches your main application Makefile.

**Reply by smpark7** [reply]
This fixed the issue. Thank you!
